apple/RNGestureHandler.h

```diff
@@ -9,6 +9,12 @@
 #import <Foundation/Foundation.h>
 #import <React/RCTConvert.h>
 
+#if TARGET_OS_OSX
+#define TargetScrollView NSScrollView
+#else
+#define TargetScrollView UIScrollView
```

Comment:
```suggestion
#define RNGHUIScrollView NSScrollView
#else
#define RNGHUIScrollView UIScrollView
```

I'd stick to the convention that's already in use (maybe dropping UI from the name, not strong opinions here), and move it to the other file with macOS compat stuff.

Reply:
I've changed it to RNGHUIScrollView. We already have RNGHUIView, so I think we can keep the UI in the name.